src/kirby/components/progress-bar/progress-bar.component.spec.ts

```diff
@@ -0,0 +1,27 @@
+/* tslint:disable:no-unused-variable */
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { DebugElement } from '@angular/core';
+
+import { ProgressBarComponent } from './progress-bar.component';
+
+describe('ProgressBarComponent', () => {
```

Comment:
Maybe we should come up with a test or two like:

- Can it handle negative numbers?
- what if no color is defined
- etc

Reply:
1. I am a bit noobish in the testing, but.... How would we test if it could handle negative values? The value is used ina binding so to test if it could handle it, we would need to test the UI right? And see that the width is not disturbed in any way.
2. If no color is defined, a default color is provided. Don't know how to test that either since it again is a UI thing :D

Reply:
1. Well maybe you need to define if negative values are allowed. And the same goes what if progress is over 100%. Does that make sense to have negative and >100 progress numbers?
2.  You are right, you would have to test the view of the component to test the default values or you would have to test that the div elements styling. It can be done. Maybe have a look on the `list` component.

I would say that 1. tests are more important then 2. tests.